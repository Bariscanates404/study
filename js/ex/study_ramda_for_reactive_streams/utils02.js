export const isEven = i => i % 2 === 0
export const square = i => i * i
