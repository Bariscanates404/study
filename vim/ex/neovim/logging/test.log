00:49:49 [INFO] name: A log message                  file="e02.lua", line=55
00:49:49 [WARN] name: A log message with keyword arguments file="e02.lua", line=56, warning="something happened"
