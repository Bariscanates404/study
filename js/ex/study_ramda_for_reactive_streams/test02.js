import test from 'ava'
import {isEven, square} from './utils02'
test('isEven', t => {
  t.true(isEven(2))
  t.true(isEven(4))
  t.false(isEven(1))
  t.false(isEven(11))
})
test('isEven snapshot', t => {
  t.snapshot([2, 4, 1, 11].map(isEven))
})

