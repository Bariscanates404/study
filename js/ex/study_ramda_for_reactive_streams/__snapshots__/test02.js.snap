// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`isEven snapshot 1`] = `
Array [
  true,
  true,
  false,
  false,
]
`;
